dpsim-villas/src/InterfaceVillasQueueless.cpp

```diff
@@ -0,0 +1,374 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
What about the copyright here?

Reply:
Sorry, I have to insist. Copyright info is missing here. This is a new contribution and as such should have a copyright header.

If the other VILLAS related code here does not have it, its a mistake. We should add it lateron too.

We should also try to harmonize the header style with the remaining DPsim code.

Reply:
I'm not sure what to do here. Some files have the SPDX note above but without copyright notice and some files only have a copyright notice with a different license and no SPDX at all. If I copy the SPDX header from VILLAS, I will introduce a third type of header.

Reply:
I'll do that anyways, because it's the most correct of all options.